Eastern-Research-Group/mywaterway
# Feature/137 update template

## Related Issues:
* HMW-137

## Main Changes:
* Updated the HTML templates to match those on the main [https://www.epa.gov]() site.
* The EPA navigation links above the HMW content are now drop-down menus.

## Steps To Test:
1. Go to [http://localhost:3000/](), and browse any site under the `mywaterway` subdomain
2. Decrease the screen width to confirm responsiveness on mobile browsers
3. Disable JavaScript to confirm that the interface adapts and the proper alerts are presented to the user

## TODO:
- [ ] I included the "return to top" button from [https://www.epa.gov](), but it might get in the way of the HMW application, so it could use some usability testing.
